src/v1/schemas/authentication.ts

```diff
@@ -0,0 +1,51 @@
+import { FastifyCookie } from '@fastify/cookie';
+import { FastifySessionObject } from '@fastify/session';
+import { Type as t } from '@sinclair/typebox';
+import { BaseResponse } from '.';
+
+export const RegisterDeviceWithWebAuthnSchema = {
+  body: t.Object({
+    passkeyName: t.Optional(t.String()),
+    username: t.String(),
+    credential: t.Object({
+      id: t.String(),
+      algorithm: t.Enum({ RS256: 'RS256', ES256: 'ES256' }),
```

Comment:
we can import this enum from prisma. Check how it was done here https://github.com/backpack-fux/pylon-v2/blob/1056dbe796f8ba44e3582ff1bb4f48bb06073f7a/src/v1/schemas/transaction.ts#L19C23-L19C43
```suggestion
      algorithm: t.Enum(),
```